xml/System.IO/WindowsRuntimeStreamExtensions.xml

```diff
@@ -203,74 +316,33 @@
         <Parameter Name="windowsRuntimeStream" Type="Windows.Storage.Streams.IInputStream" RefType="this" />
       </Parameters>
       <Docs>
-        <param name="windowsRuntimeStream">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
-      </Docs>
-    </Member>
-    <Member MemberName="AsStreamForRead">
-      <MemberSignature Language="C#" Value="public static System.IO.Stream AsStreamForRead (this Windows.Storage.Streams.IInputStream windowsRuntimeStream, int bufferSize);" />
-      <MemberSignature Language="ILAsm" Value=".method public static hidebysig class System.IO.Stream AsStreamForRead(class Windows.Storage.Streams.IInputStream windowsRuntimeStream, int32 bufferSize) cil managed" />
-      <MemberSignature Language="DocId" Value="M:System.IO.WindowsRuntimeStreamExtensions.AsStreamForRead(Windows.Storage.Streams.IInputStream,System.Int32)" />
```

Comment:
@v-kents not sure at what point but it seems that there a few APIs that were dropped from this file. Can you please add them back and port the content for them as well? Other than that, I've made all the other fixes needed.

Reply:
Looks like they got clipped very early on (due to a cross-eyed me). Sorry about that. Added back.